tests/components/bayesian/test_binary_sensor.py

```diff
@@ -436,83 +493,171 @@ async def test_multiple_observations(hass: HomeAssistant) -> None:
     # Calculated using bayes theorum where P(A) = 0.2, P(B|A) = 0.2, P(B|notA) = 0.6
 
     assert state.state == "off"
-    assert state.attributes.get("observations")[0]["platform"] == "multi_state"
-    assert state.attributes.get("observations")[1]["platform"] == "multi_state"
+    assert state.attributes.get("observations")[0]["platform"] == "state"
+    assert state.attributes.get("observations")[1]["platform"] == "state"
+
 
+async def test_multiple_numeric_observations(
+    hass: HomeAssistant, issue_registry: ir.IssueRegistry
+) -> None:
+    """Test sensor on numeric state platform observations with more than one range.
 
-async def test_multiple_numeric_observations(hass: HomeAssistant) -> None:
-    """Test sensor with multiple numeric observations of same entity."""
+    This tests an example where the probability of it being a 'nice day' varies over
+    a series of temperatures. Since this is a multi-state, all the non-observed ranges
+    should be ignored and only the range including the observed value should update
+    the prior. When a value lands on above or below (15 is tested) it is included if it
+    equals `below`, and ignored if it equals `above`.
+    """
 
     config = {
         "binary_sensor": {
             "platform": "bayesian",
-            "name": "Test_Binary",
+            "name": "nice_day",
             "observations": [
                 {
                     "platform": "numeric_state",
-                    "entity_id": "sensor.test_monitored",
+                    "entity_id": "sensor.test_temp",
+                    "below": 0,
+                    "prob_given_true": 0.05,
+                    "prob_given_false": 0.2,
+                },
+                {
+                    "platform": "numeric_state",
+                    "entity_id": "sensor.test_temp",
                     "below": 10,
                     "above": 0,
-                    "prob_given_true": 0.4,
-                    "prob_given_false": 0.0001,
+                    "prob_given_true": 0.1,
```

Comment:
can you share the reason to change the values ?

Reply:
A few reasons:
1. Using extreme values like `0.0001` tends to give extreme results that are bunched up to the limits of the probability distribution (near `0` or `1`). ![image](https://github.com/user-attachments/assets/a0cd5662-1db9-4c9b-b1b8-bc701a3b1d5b)
  - This makes the results less sensitive to minor changes in how calculations are done, which may mean a breaking change slips through the net (we only check results to 2 decimal places), minor changes will cause the biggest shift in outputs with values near to `0.5`
 2. I wanted to change to config to be about something tangible ("e.g is it a nice day?") to improve readability and real-world applicability.
 3. Extreme values very near `0` or `1` are [discouraged in the documentation](https://www.home-assistant.io/integrations/bayesian/#estimating-probabilities).